internal/netconf/tpl/networkd/10-lan.network.tpl

```diff
@@ -1,7 +1,9 @@
-{{- /*gotype: github.com/metal-stack/metal-networker/internal/netconf.SystemdNetworkData*/ -}}
+{{- /*gotype: github.com/metal-stack/metal-networker/internal/netconf.LanNetworkData*/ -}}
```

Comment:
You renameed "VlanID" into "VLANID" so the consequence is to name LanNetworkData -> "LANNetworkData".